definitions/infra-container/definition.yml

```diff
@@ -39,6 +52,11 @@ goldenTags:
 - environment
 - container.state
 
+dashboardTemplates:
+  opentelemetry/otelcol/dockerstatsreceiver:
```

Comment:
I'm not sure if having the name with the `/` character will work. According to the docs this should be `{provider}/{name}`

Reply:
also the `instrumentation.name` is not a metric attribute but a [renamed tag](https://github.com/newrelic/entity-definitions/pull/1096/files#diff-b6e5a14d10a6462e013978ff92f0dee9fc5c630fc3229e7fc38fd263dadc187fR36). Would this work?